src/client/pfs/pfs.proto

```diff
@@ -561,6 +561,16 @@ message DiffFileResponseV2 {
   FileInfo new_file = 2;
 }
 
+message CreateTempFileSetResponse {
+  string fileset_id = 1;
+  int64 expires_at = 2;
```

Comment:
After looking over this a bit more, I'm not sure if returning the expire time is that useful and it requires quite a bit of threading through the storage layers. I guess you could use this information to handle renewal for the rare case where a response is really delayed, but I think most use cases will probably just have a fixed renewal based on preexisting knowledge of the default ttl.

Reply:
I like the lease renewal algorithm that works as follows: Get a lease, check when it expires, schedule a renewal halfway to the expire time, repeat.  This allows the client to know if it has a lease that it can't work with (because it expires too quickly).